src/Drivers/nlpMDS_raja_ex4.cpp

```diff
@@ -353,14 +387,16 @@ bool Ex4::eval_cons(const size_type& n, const size_type& m,
 
   /// @todo This is parallelizing only 3 iterations
   // inequality constraints
-  if(num_cons == 3 && haveIneq_)
-  {
+  if(num_cons == 3 && haveIneq_) {
     RAJA::forall<ex4_raja_exec>(RAJA::RangeSegment(0, num_cons),
       RAJA_LAMBDA(RAJA::Index_type irow)
       {
-        const int con_idx = (int) idx_cons[irow];
+        assert(ns>=0);
+        const int con_idx = (int) idx_cons_in[irow];
+        assert(con_idx>=0);
         assert(con_idx < ns+3);
-        const int conineq_idx=con_idx - ns;
+        const int conineq_idx = con_idx - ns;
```

Comment:
should we use `index_type` inside RAJA?

Reply:
I think converting explicitly to `int` is safer (down the road)...